infra/modules/aws-psoxy-bulk/main.tf

```
@@ -98,31 +81,55 @@ resource "aws_s3_bucket" "output" {
   bucket = "psoxy-${var.instance_id}-output"
 }
 
-module "psoxy-file-handler" {
-  source = "../aws-psoxy-instance"
+module "psoxy_lambda" {
```

```suggestion
module "psoxy-lambda" {
```
modules use dashusually